lib/collections.js

```diff
@@ -47,7 +82,24 @@ longAndShort['longest'] = '';
 longAndShort['shortest'] = paragraph;
 
 /* your code starts here */
-
+var j = 0;
```

Comment:
It's actually not even necessary to use these variables to keep track of the lengths of words; we can just write `longAndShort['longest'].length` to get the length of the current longest (or shortest) word.